napari/plugins/hub.py

```diff
@@ -8,15 +8,17 @@
 from typing import Generator, Optional
 from urllib import error, request
 
-from .utils import normalized_name, ProjectInfo
+from .utils import ProjectInfo, normalized_name
```

Comment:
btw, since we depend on npe2 now, it might be nice to use `npe2.manifest.package_metadata.PackageMetadata`.  See, for example, `PackageMetadata.for_package("napari")`

Reply:
👍🏼

Reply:
What should I use for `metadata_version` ?

Reply:
use the lowest version from https://packaging.python.org/en/latest/specifications/core-metadata/#project-url-multiple-use that contains all the fields you want.  So, if you used ` name, version, home_page, summary, author, and license`, you could use `metadata_version='1.0'` (since those were all present then).  Also from that page, looks like home_page really is the "primary" url that matches what we use it for here

Reply:
https://github.com/napari/npe2/pull/96 will make this optional